### PR TITLE
l10n update for occitan language

### PR DESCRIPTION
**Pull request by Quent-in**
Following this issue : https://github.com/tootsuite/mastodon/issues/3399
I made the changes required to display the dates in full occitan.
I hope it is alright now.

### CODE REVIEW

File: config/locales/oc.yml [hunk 1]
```diff
@@ -206,8 +206,62 @@ oc:
   authorize_follow:
     error: O planhèm, i a agut una error al moment de cercar lo compte
     follow: Sègre
-    prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
```
Comment (ykzts) [inline config/locales/oc.yml]: why remove?

File: config/locales/oc.yml [hunk 2]
```diff
@@ -206,8 +206,62 @@ oc:
...
+    prompt_html: 'Avètz demandat de sègre :'
     title: Sègre %{acct}
+  date:
+
```
Comment (ykzts) [inline config/locales/oc.yml]: remove blank line.

**Review by ykzts — DISMISSED** [review]
edit [`i18n-tasks.yml`](https://github.com/tootsuite/mastodon/blob/7c7c18fdea52ce288a87d2455f1efc55f74af003/config/i18n-tasks.yml#L34-L57).

File: config/locales/oc.yml (hunk 1 above)
Comment (Quent-in) [inline config/locales/oc.yml]: Well, normally the subject of the verb is included in the conjugation, I wondered if I could put the emphasis on the "avètz" instead.

Comment (ykzts) [inline config/locales/oc.yml]: `<strong>%{self}</strong>` is an important thing that shows user information. Do not want to delete it as much as possible

Comment (Quent-in) [inline config/locales/oc.yml]: I've just put it back, I hope it's okay now.

**Review by Gargron — APPROVED** [review]

File: config/locales/oc.yml [hunk 3]
```diff
@@ -208,6 +208,59 @@ oc:
     follow: Sègre
     prompt_html: 'Avètz (<strong>%{self}</strong>) demandat de sègre :'
     title: Sègre %{acct}
+  date:
```
Comment (Gargron) [inline config/locales/oc.yml]: Judging by the travis failure, date keys might be indented wrongly?

**Review by Gargron — CHANGES_REQUESTED** [review]

File: config/locales/oc.yml (hunk 3 above)
Comment (ykzts) [inline config/locales/oc.yml]: No. Need to change `config/i18n-tasks.yml`. see #3572

**Review by Gargron — APPROVED** [review]

**Review by ykzts — APPROVED** [review]

**Pull request merged**